pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
```

Comment:
Got it. So lets try to make the description a bit clearer. Try to explain what a "Yesod scaffolding" is. Think of it from a newcomer's point of view.